modules/sdk-coin-eth/src/eth.ts

```diff
@@ -2096,13 +2096,13 @@ export class Eth extends BaseCoin {
    * @param typedData the typed data to prepare
    * @return a buffer of the result
    */
-  encodeTypedData(typedData: TypedData<never>): Buffer {
```

Comment:
this is a breaking change

Reply:
resolving this request since this feature are not live even though they are breaking changes